spec/functional/account.spec.js

```diff
@@ -46,8 +46,8 @@ describe("account", () => {
             });
         }).then((response) => {
             expect(response.statusCode).toBe(200);
-            expect(response.headers["Content-Type"]).toBe("text/plain");
-            expect(response.body.toString("binary")).toBe("This is a test token");
+            expect(response.getHeader("Content-Type")).toBe("text/plain");
+            expect(response.body.toString()).toBe("This is a test token\n");
```

Comment:
This looks wrong. The body of the request (line 30) does not have a newline but this response includes a newline. That seems to indicate OpenToken is adding a newline to the payload instead of keeping it verbatim, and that's incorrect behavior.

This same feedback applies to the addition of a newline on line 84.